apps/modernization-ui/src/apps/page-builder/page/management/edit/section/Section.tsx

```diff
@@ -11,13 +11,15 @@ import './manage/ManageSectionModal.scss';
 import { AddSubSection } from '../subsection/manage/AddSubSection';
 import { AlertInLineProps } from './manage/ManageSectionModal';
 import { ManageSubsection } from '../subsection/manage/ManageSubsection';
+import { authorization } from 'authorization';
 
 type Props = {
     section: PagesSection;
     onAddQuestion: (subsection: number) => void;
     onEditQuestion: (question: PagesQuestion) => void;
     handleDeleteSection: () => void;
     addQuestionModalRef: RefObject<ModalRef>;
+    handleStatus: () => void;
```

Comment:
Suggestion: Would be good to standardize on how we name callbacks.
I tend to favor the `prop` callback to be a `onAction`, such as `onStatusChange` or `onContentChange` with the function that resolves the callback using the `handleAction` naming convention.